server/controllers/UserController.js

```diff
@@ -66,18 +73,15 @@ class UserController {
    */
   static login(req, res, next) {
     if (!req.body.user) {
-      return res.status(422)
-        .json({ errors: { user: 'must be an object' } });
+      return res.status(422).json({ errors: { user: 'must be an object' } });
     }
 
     if (!req.body.user.email) {
-      return res.status(422)
-        .json({ errors: { email: 'can\'t be blank' } });
+      return res.status(422).json({ errors: { email: 'can\'t be blank' } });
```

Comment:
use template literals

Reply:
Since there is no line break in the string, nor a javaScript code, eslint is flagging the use of template literals as inappropriate.